docs/70-jobs.md

```diff
@@ -0,0 +1,172 @@
+
+### Job.yaml
+
+Each Job has one or more applications given as a List in Job.yaml. Applications are defined using the following properties:
+```yaml
+  applications:
+    - name: 
+      application: 
+      application_args: 
+        - 
+      main_class: 
+      jars: 
+        - 
+      py_files: 
+        - 
+      files:
+        - 
+      driver_java_options: 
+        - 
+      driver_library_path: 
+      driver_class_path: 
+      driver_memory: 
+      executor_memory: 
+      driver_cores: 
+      executor_cores: 
+```
+_Please note: the only required fields are name and application. All other fields may be removed or left blank._
+
+Jobs also require a definition of the cluster on which the Applications will run. The following properties define a cluster:
+```yaml
+  cluster_configuration:
+    vm_size: <the Azure VM size>
+    size: <the number of nodes in the Cluster>
+    docker_repo: <Docker Image to download on all nodes>
+    custom_scripts: 
+      - List
+      - of
+      - paths
+      - to
+      - custom
+      - scripts
+```
+_Please Note: For more information about Azure VM sizes, see [Azure Batch Pricing](https://azure.microsoft.com/en-us/pricing/details/batch/). And for more information about Docker repositories see [Docker](./12-docker-iamge.md)._
+
+_The only required fields are vm_size and either size or size_low_pri, all other fields can be left blank or removed._
+
+A Job definition may also include a default Spark Configuration. The following are the properties to define a Spark Configuration:
+```yaml
+  spark_configuration:
+    spark_defaults_conf: </path/to/your/spark-defaults.conf>
+    spark_env_sh: </path/to/your/spark-env.sh>
+    core_site_xml: </path/to/your/core-site.xml>
+```
+_Please note: including a Spark Configuration is optional. Spark Configuration values defined as part of an application will take precedence over the values specified in these files._
+
+
+Below we will define a simple, functioning job definition.
+```yaml
+# Job Configuration
+
+job:
+  id: test-job
+  cluster_configuration:
+    vm_size: standard_f2
+    size: 3
+ 
+  applications:
+    - name: pipy100
+      application: /path/to/pi.py
+      application_args: 
+        - 100
+    - name: pipy200
+      application: /path/to/pi.py
+      application_args: 
+        - 200
+```
+Once submitted, this Job will run two applications, pipy100 and pipy200, on an automatically provisioned Cluster with 3 dedicated Standard_f2 size Azure VMs. Immediately after both pipy100 and pipy200 have completed the Cluster will be destroyed. Application logs will be persisted and available.
+
+### Commands
+Submit a Spark Job:
+
+```sh
+aztk spark job submit --id <your_job_id> --job-conf </path/to/job.yaml>
+```
+
+NOTE: The Job id (`--id`) can only contain alphanumeric characters including hyphens and underscores, and cannot contain more than 64 characters. Each cluster **must** have a unique cluster id.
```

Comment:
Do the application names have the same requirement? I believe they do - if so, worth mentioning here.